gen/common/util.mli

```diff
@@ -32,7 +32,38 @@ val arch_opt : Archs.t ref -> spec
 
 val parse_cmdline : spec list -> (string -> unit) -> unit
 
+module type Monad = sig
+  type 'a t
+  val pure : 'a -> 'a t
```

Comment:
Not sure if it worths to use the function name from Ocaml official tutorial (https://ocaml.org/docs/monads), namely, `return` rather than `pure`.

Reply:
Good point, will change. `pure` is often how `return` is called for "applicative functors" and you can often find it as an [alias of `return`](https://ocaml.org/p/containers/latest/doc/containers/CCOption/index.html#val-pure). But I agree with you here.